gpflow/expectations.py

```diff
@@ -492,9 +492,15 @@ def _expectation(p, kern, feat, none2, none3):
     return functools.reduce(tf.add, [_expectation_fn(k) for k in kern.kern_list])
 
 
-@dispatch((Gaussian, MarkovGaussian),
-            kernels.Sum, InducingPoints,
-            (mean_functions.Linear, mean_functions.Constant), type(None))
+@dispatch(Gaussian,
```

Comment:
What was the particular reason to split up these two?

Reply:
I split it because there is no implementation for Linear and Constant mean functions with the MarkovGaussian distribution, and it wasn't falling back to the quadrature. Happy to revert it back if I misunderstood.

Reply:
This is correct.